### PR TITLE
Support Windows 10 for Android build

### PR DESCRIPTION
**Pull request by rtsisyk**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -187,12 +187,14 @@ include(BundledBoost)
 
 # Scripts
 
-execute_process(
-    COMMAND "${OMIM_ROOT}/tools/unix/check_cert.sh"
-    RESULT_VARIABLE CheckCertResult
-)
-if (CheckCertResult)
-    message(FATAL_ERROR "Certificate check failed")
+if (NOT CMAKE_HOST_WIN32)
```
Comment (biodranik) [inline CMakeLists.txt:190]: 64-битный windows хост тоже сработает?

File: docs/INSTALL.md [hunk 2]
```diff
@@ -240,13 +247,28 @@ or with private repository
...
+1. Install Visual Studio 2019 Community Edition.
+2. Add cl.exe to your PATH (`C:\\Program Files (x86)\\Microsoft Visual Studio\\2019\\Community\\VC\\Auxiliary\\Build\\vcvars32.bat`).
+3. Run `./configure.sh` from *Git (for Window) Bash* and ignore all errors related to Boost.
+4. Go to `./3party/boost`, run `./bootstrap.bat`, and then `b2 headers` to configure Boost.
+
 Set Android SDK and NDK path:
 
 ```bash
 # Linux
 ./tools/android/set_up_android.py --sdk $HOME/Android/Sdk
```
Comment (biodranik) [inline docs/INSTALL.md:247]: Эти скрипты разве до сих пор нужны?

**Review by biodranik — APPROVED** [review]
LGTM with comments

File: docs/INSTALL.md (hunk 2 above)
Comment (rtsisyk) [reply docs/INSTALL.md:247]: Попробовал, нужны:

>SDK location not found. Define location with an ANDROID_SDK_ROOT environment variable or by setting the sdk.dir path in your project's local properties file at 'organicmaps/android/local.properties'.

File: CMakeLists.txt (hunk 1 above)
Comment (rtsisyk) [reply CMakeLists.txt:190]: Да, на ней и тестируем.

https://cmake.org/cmake/help/latest/variable/CMAKE_HOST_WIN32.html:

>True if the host system is running Windows, including Windows 64-bit and MSYS.

**Pull request merged**